.github/workflows/ci.yml

```diff
@@ -48,7 +48,7 @@ jobs:
 
       - uses: actions/setup-node@v4
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
Let's make sure @nmacgreg is aware this is part of the deployment in the release notes.